UI/window-basic-main.cpp

```diff
@@ -6648,8 +6651,14 @@ void OBSBasic::BroadcastButtonClicked()
 					QTStr("Output.BroadcastStartFailed"),
 					last_error, true);
 				ui->broadcastButton->setChecked(false);
+				// broadcast start failed event
```

Comment:
```suggestion
```
This comment is not meaningful because the event name describes the same thing.

Reply:
changed and submitted.